x-pack/plugins/security_solution/public/timelines/components/certificate_fingerprint/index.test.tsx

```diff
@@ -47,15 +43,11 @@ describe('CertificateFingerprint', () => {
         />
       </TestProviders>
     );
-    expect(
-      removeExternalLinkText(
-        wrapper.find('[data-test-subj="certificate-fingerprint-link"]').first().text()
-      )
-    ).toContain('3f4c57934e089f02ae7511200aee2d7e7aabd272');
+    expect(screen.getByText('3f4c57934e089f02ae7511200aee2d7e7aabd272')).toBeInTheDocument();
```

Comment:
I wonder if there are Kibana guidelines on using `screen.getByText` vs `render.getByText`? I have seen both in the codebase, so I assume there is no such guideline.

In this PR I'm going with the `screen`-approach as it's recommended by the `testing-library` maintainer (see https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)